mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Realtime.MixProject do
   def project do
     [
       app: :realtime,
-      version: "2.58.0",
+      version: "2.59.1",
```

Comment:
2.58.2 I think? Might need to rebase/merge with main first?

Reply:
no, the presence disable was set as minor vs patch 👀

Reply:
Oh I see 😆